src/types.js

```diff
@@ -54,7 +54,7 @@ export const types = {
           align: ['stretch', 'start', 'center', 'end'],
           direction: ['column', 'row', 'row-responsive'],
           gap: ['xsmall', 'small', 'medium', 'large', 'xlarge'],
-          justify: ['between', 'start', 'center', 'end'],
+          justify: ['around', 'between', 'start', 'center', 'end', 'evenly'],
```

Comment:
I'm thinking `stretch` value is also missing, it should be the default, maybe we should replace it with the current `undefined` that is shown on the drop options

Reply:
Sorry for not adding it, it is also missing from the Box typescript file so I was not sure about it :)

Reply:
@sofiageo no worries buddy, thanks for the PR!